### PR TITLE
Settings for Powheg V2 bbbar4l emu/mue production

### PR DESCRIPTION
**Pull request by jfernan2**

### CODE REVIEW

**Comment by agrohsje** [comment]
Hi Javi,

can you update with this PR the res version in run_pwg.py : 
if [ "$process" = "b_bbar_4l" ]; then                                                                                                                                                                                
  export POWHEGSRC=powhegboxRES_rev3463_date20171102.tar.gz                                                                                                                                                          
fi    
This is the latest one, I added the new tar ball and used it for these gridpacks.                                                                                                                                                                                                               
Thanks, Alexander.

**Comment by agrohsje** [comment]
Hi Javi,

these lines are automatically added by the scripts: 
+rwl_file 'pwg-rwl.dat'
+rwl_format_rwgt 1
They could go?

Cheers, Alexander.

**Comment by efeyazgan** [comment]
@agrohsje could this be merged now?

File: bin/Powheg/production/2017/13TeV/bbbar_4l/powheg.input [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ih2   1                                          ! hadron 2 (1 for protons, -1 for antiprotons)
+ebeam1 6500                                      ! energy of beam 1
+ebeam2 6500                                      ! energy of beam 2
+lhans1 306000                                    ! pdf set for hadron 1 (LHA numbering) [if using LHA pdfs]
+lhans2 306000                                    ! pdf set for hadron 2 (LHA numbering) [if using LHA pdfs]
+
+use-old-grid 1                                   ! if 1 use old grid if file pwggrids.dat is present (<> 1 regenerate)
+use-old-ubound 1                                 ! if 1 use norm of upper bounding function stored in pwgubound.dat, if present; <> 1 regenerate
+
+ncall1 80000                                     ! number of calls for initializing the integration grid
```
Comment (perrozzi) [inline bin/Powheg/production/2017/13TeV/bbbar_4l/powheg.input]: this naively looks like a low number of iterations, have you looked at the smoothness of the integration grids?
and/or are you using the parallel gridpack production?

**Comment by agrohsje** [comment]
Hi Luca,

I used parallel gridpack production with 64 jobs. 

Cheers, Alexander.

**Pull request merged**